src/Raven.Server/Integrations/PostgreSQL/CsvToPg.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System;
+using System.Collections.Generic;
+using Microsoft.VisualBasic.FileIO;
+using Raven.Server.Integrations.PostgreSQL.Messages;
+
+namespace Raven.Server.Integrations.PostgreSQL
+{
+    public static class CsvToPg
```

Comment:
What is this for?

Reply:
This is used to handle the "hardcoded queries" about pg types, chracters set, table schema etc. The responses are generated based on results exported as CSV from PgAdmin as described here:

https://issues.hibernatingrhinos.com/issue/RavenDB-16880#focus=Comments-67-91291.0-0